README.md

```diff
@@ -1,70 +1,48 @@
 
-![](./branding/logo/PNG/96-logo-OnlySymbol.png)
+![](./branding/logo/PNG/96-logo-horizontal.png)
 
 # Stylable - CSS for Components
 
-> "New ideas will come along, but they will extend CSS rather than replace it. I believe that the CSS code we write today will be readable by computers 500 years from now."
-
-[Håkon Wium Lie](https://dev.opera.com/articles/css-twenty-years-hakon/), co-creator of CSS
-
-At Wix, we agree. We love CSS. It's simple, declarative syntax that is native in browsers and is easily the fastest way to add styles to web pages and web apps. But when writing CSS that is scoped to individual components, developers have to maintain highly-specific selectors, using elaborate conventions to fake namespacing. Writing and maintanining CSS across large teams and large projects can be tricky.
-
-We want to extend CSS so that it is easier to use in a component ecosystem, but without losing the aspects of CSS that make it great: declarative, familiar, static and fast. So we created **Stylable**.
-
-**Styable** is a preprocessor that allows you to write style rules in CSS syntax, with some extensions that we believe adhere to the spirit of CSS.
-
-So what does **Stylable** do?
+**Stylable** enables you to build reusable, highly-performant components. Each component exposes a style API that maps its internal parts so you can reuse components across teams without sacrificing stylability.
 
 * Scopes styles to components so they don't "leak" and clash with other styles.
```

Comment:
* Scopes styles to components so they don't "leak" and clash with other styles.
* Enable custom  pseudo-classes and pseudo-elements  that abstract the internal structure of a component so that it can be styled externally.
* Uses themes so you can apply a different look and feel across your web application.

Reply:
I like what you had in the email

* Scopes styles to components so they don't "leak" and clash with other styles.
* Provides a style API to enable abstracting the internal structure of a component so that it can be styled externally.
* Uses themes so you can apply a different look and feel to the same components across your web application.
* Enables you to create, reuse and style pseudo-classes and pseudo-elements.

Reply:
Used Ido's version